src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
 }
 
 .hoc {
-    .title-banner {
-        &.wbb-bg {
-            background-image: url("/images/hoc2015/hide-bg.jpg");
+    .img,
+    .button {
+        &.card-info {
```

Comment:
From https://gist.github.com/bobbygrace/9e961e8982f42eb91b80#8-miscellany
> Explicitly write out class names in selectors. Don’t concatenate strings or use preprocessor trickery to build a class name. We want to be able to search for class names and that makes it impossible.

Avoid the `&` for this type of thing so our css is as flat as possible.  We might need to adjust the sass linter rules, or disable the sass linter altogether until we've transitioned.